react/hooks/useShippingActions.js

```diff
@@ -0,0 +1,132 @@
+import { useCallback, useEffect, useState } from 'react'
+import { usePixel, usePixelEventCallback } from 'vtex.pixel-manager'
+import { useSSR } from 'vtex.render-runtime/react/components/NoSSR'
+
+import useShouldDisableFacet from './useShouldDisableFacet'
+
+const shippingActionTypes = {
```

Comment:
If those guys are constants I believe that is better to name them UPPER_CASED. Like the SHIPPING_OPTIONS.

Reply:
When exploring VTEX code  I usually see  UPPER_CASE for constant primitive types and camelCase for constant objects. For example, I usually see

```
const config = {port: 5000,  apiEndPoint: 'api.example.com'}
```

Instead of

```
const CONFIG = {port: 5000,  apiEndPoint: 'api.example.com'}
```

I don't see any of them being better or most recommended by VTEX developers. I think both are ok as long as we keep consistency.

Reply:
I think to maintain consistency we can change SHIPPING_OPTIONS to use camel case bacause it is a constant object. 
I think it's better because we already have this pattern being used in other objects, just the SHIPPING_OPTIONS is in another pattern.